src/k8s-job-exit-spec/config/k8s-job-exit-spec.yaml

```diff
@@ -804,6 +811,7 @@ spec:
   pattern:
     runtimeContainerPatterns:
     - userLogRegex: "(?msi)tensorflow.*ResourceExhaustedError.*OOM.*"
+    - userLogRegex: "ran out of memory trying to allocate"
```

Comment:
This is also GPU OOM in most cases, so it is not the same kind as "ran out of memory trying to allocate"

Reply:
Why remove "ran out of memory trying to allocate"

---
In reply to: [353554911](https://github.com/microsoft/pai/pull/3959#discussion_r353554911) [](ancestors = 353554911)